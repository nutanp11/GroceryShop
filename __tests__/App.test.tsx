import { render } from '@testing-library/react-native';
import App from '../App';

test('Snapshot', ()=>{
  const { toJSON } = render(<App/>);
  expect(toJSON()).toMatchSnapshot();
})
