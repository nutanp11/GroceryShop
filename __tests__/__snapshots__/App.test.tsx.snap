// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<View
  style={
    {
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView
    style={
      {
        "backgroundColor": "#ffffff",
        "flex": 1,
      }
    }
  >
    <RNCSafeAreaProvider
      onInsetsChange={[Function]}
      style={
        [
          {
            "flex": 1,
          },
          undefined,
        ]
      }
    >
      <RNSScreenStack
        style={
          {
            "flex": 1,
          }
        }
      >
        <RNSScreen
          accessibilityElementsHidden={false}
          activityState={2}
          collapsable={false}
          fullScreenSwipeShadowEnabled={true}
          gestureResponseDistance={
            {
              "bottom": -1,
              "end": -1,
              "start": -1,
              "top": -1,
            }
          }
          hasLargeHeader={false}
          importantForAccessibility="auto"
          nativeBackButtonDismissalEnabled={false}
          onAppear={[Function]}
          onDisappear={[Function]}
          onDismissed={[Function]}
          onGestureCancel={[Function]}
          onHeaderBackButtonClicked={[Function]}
          onHeaderHeightChange={[Function]}
          onNativeDismissCancelled={[Function]}
          onSheetDetentChanged={[Function]}
          onTransitionProgress={[Function]}
          onWillAppear={[Function]}
          onWillDisappear={[Function]}
          replaceAnimation="push"
          sheetAllowedDetents={
            [
              1,
            ]
          }
          sheetCornerRadius={-1}
          sheetElevation={24}
          sheetExpandsWhenScrolledToEdge={true}
          sheetGrabberVisible={false}
          sheetInitialDetent={0}
          sheetInitialDetentIndex={0}
          sheetLargestUndimmedDetent={-1}
          sheetLargestUndimmedDetentIndex={-1}
          stackPresentation="push"
          style={
            [
              [
                {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                undefined,
              ],
              {
                "zIndex": undefined,
              },
            ]
          }
          swipeDirection="horizontal"
        >
          <RNSScreenContentWrapper
            collapsable={false}
            style={
              [
                {
                  "flex": 1,
                },
                [
                  {
                    "backgroundColor": "rgb(242, 242, 242)",
                  },
                  undefined,
                ],
              ]
            }
          >
            <RCTScrollView
              contentContainerStyle={
                {
                  "flexGrow": 1,
                }
              }
            >
              <View>
                <View
                  style={
                    {
                      "backgroundColor": "#ffffff",
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      {
                        "flex": 0.8,
                        "padding": 10,
                      }
                    }
                  >
                    <View>
                      <Text
                        style={
                          {
                            "alignSelf": "center",
                            "fontFamily": "Nunito-Bold",
                            "fontSize": 54,
                            "fontWeight": "900",
                          }
                        }
                      >
                        Basket
                      </Text>
                    </View>
                    <Image
                      source={
                        {
                          "testUri": "../../../src/assets/images/signin.png",
                        }
                      }
                      style={
                        {
                          "alignSelf": "center",
                          "height": 251,
                          "marginVertical": 10,
                          "width": 256,
                        }
                      }
                    />
                    <Text
                      style={
                        {
                          "alignSelf": "center",
                          "fontFamily": "Nunito-Bold",
                          "fontSize": 24,
                          "fontWeight": "900",
                        }
                      }
                    >
                      Log In
                    </Text>
                    <Text
                      style={
                        {
                          "alignSelf": "center",
                          "color": "#999999",
                          "fontFamily": "Nunito-Regular",
                          "fontSize": 14,
                          "fontWeight": "600",
                          "lineHeight": 35,
                        }
                      }
                    >
                      Please type your information
                    </Text>
                    <View
                      style={
                        [
                          {
                            "alignSelf": "center",
                            "backgroundColor": "#F4F4F4",
                            "borderRadius": 5,
                            "marginVertical": 10,
                            "padding": 5,
                            "width": "90%",
                          },
                          undefined,
                        ]
                      }
                    >
                      <TextInput
                        keyboardType="default"
                        onChangeText={[Function]}
                        placeholder="Email address"
                        placeholderTextColor="#000000"
                        style={
                          {
                            "color": "#000000",
                            "fontFamily": "Nunito-Bold",
                            "fontSize": 14,
                            "fontWeight": "900",
                            "padding": 15,
                          }
                        }
                        value=""
                      />
                    </View>
                    <View
                      style={
                        [
                          {
                            "alignSelf": "center",
                            "backgroundColor": "#F4F4F4",
                            "borderRadius": 5,
                            "marginVertical": 10,
                            "padding": 5,
                            "width": "90%",
                          },
                          undefined,
                        ]
                      }
                    >
                      <TextInput
                        keyboardType="default"
                        onChangeText={[Function]}
                        placeholder="Password"
                        placeholderTextColor="#000000"
                        style={
                          {
                            "color": "#000000",
                            "fontFamily": "Nunito-Bold",
                            "fontSize": 14,
                            "fontWeight": "900",
                            "padding": 15,
                          }
                        }
                        value=""
                      />
                    </View>
                  </View>
                  <View
                    style={
                      {
                        "flex": 0.2,
                        "justifyContent": "space-between",
                      }
                    }
                  >
                    <View>
                      <View
                        accessibilityState={
                          {
                            "busy": undefined,
                            "checked": undefined,
                            "disabled": undefined,
                            "expanded": undefined,
                            "selected": undefined,
                          }
                        }
                        accessibilityValue={
                          {
                            "max": undefined,
                            "min": undefined,
                            "now": undefined,
                            "text": undefined,
                          }
                        }
                        accessible={true}
                        collapsable={false}
                        focusable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          {
                            "alignItems": "center",
                            "alignSelf": "center",
                            "backgroundColor": "#2F2E41",
                            "borderRadius": 10,
                            "opacity": 1,
                            "padding": 10,
                            "width": "80%",
                          }
                        }
                      >
                        <Text
                          style={
                            {
                              "alignSelf": "center",
                              "color": "#FFFFFF",
                              "fontFamily": "Nunito-Bold",
                              "fontSize": 24,
                              "fontWeight": "900",
                              "lineHeight": 45,
                            }
                          }
                        >
                          Login
                        </Text>
                      </View>
                    </View>
                    <View
                      style={
                        {
                          "alignSelf": "center",
                          "flexDirection": "row",
                        }
                      }
                    >
                      <Text
                        style={
                          {
                            "alignSelf": "center",
                            "color": "#999999",
                            "fontFamily": "Nunito-Regular",
                            "fontSize": 14,
                            "fontWeight": "600",
                          }
                        }
                      >
                        New user? 
                      </Text>
                      <Text
                        onPress={[Function]}
                        style={
                          {
                            "fontFamily": "Nunito-SemiBold",
                            "fontSize": 14,
                          }
                        }
                      >
                        Create an account
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </RCTScrollView>
          </RNSScreenContentWrapper>
          <RNSScreenStackHeaderConfig
            backButtonInCustomView={false}
            backTitleFontFamily="System"
            backTitleVisible={true}
            backgroundColor="rgb(255, 255, 255)"
            color="rgb(0, 122, 255)"
            direction="ltr"
            disableBackButtonMenu={false}
            hidden={true}
            hideBackButton={false}
            largeTitleFontFamily="System"
            largeTitleFontWeight="700"
            largeTitleHideShadow={false}
            pointerEvents="box-none"
            style={
              {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "space-between",
                "position": "absolute",
                "width": "100%",
              }
            }
            title="SignIn"
            titleColor="rgb(28, 28, 30)"
            titleFontFamily="System"
            titleFontWeight="600"
            topInsetEnabled={false}
            translucent={false}
          />
        </RNSScreen>
      </RNSScreenStack>
    </RNCSafeAreaProvider>
  </RCTSafeAreaView>
</View>
`;
